security_considerations.md

```diff
@@ -10,40 +10,55 @@ if an OEM or supplier deems it necessary. In this section, we discuss several of
+ECU identifiers (e.g., serial numbers) specified in the Targets metadata signed by the Director repository do not solve this problem, because: (1) they are used by the Director repository only to instruct which ECU should install which image, and (2) they are not specified in the Targets metadata signed on the Image repository, because it is impractical to list all ECU identifiers that pertain to an image.
 
-In order to avoid this problem, the targets metadata about unencrypted images on the image repository SHOULD always include the TargetsModule.Custom.hardwareIdentifier attribute. A hardware identifier allows an OEM and / or its suppliers to succinctly capture an entire class of ECUs without listing their ECU identifiers one by one. Note that the OEM and / or its suppliers SHALL ensure that hardware identifiers are unique across different hardware types of ECUs, so that attackers who compromise the director repository cannot cause ECUs of one type to install images intended for another type.
+In order to avoid this problem, the Targets metadata about unencrypted images on the Image repository SHOULD always include the TargetsModule.Custom.hardwareIdentifier attribute. A hardware identifier allows an OEM and / or its suppliers to succinctly capture an entire class of ECUs without listing each of their ECU identifiers. Note that the OEM and / or its suppliers SHALL ensure that hardware identifiers are unique across different hardware types of ECUs, so that attackers who compromise the Director repository cannot cause ECUs of one type to install images intended for another type.
 
-Before an ECU installs a new image, it SHOULD always check the hardware type of the image so that attackers cannot cause ECUs to install randomly chosen images that have not been intended for it.
+Before an ECU installs a new image, it SHOULD always check the hardware type of the image so that attackers cannot cause any ECU to install randomly chosen images not intended for it.
 
-## Preventing rollback attacks in case of director compromise
+## Preventing rollback attacks in case of Director compromise
 
-On the Exceptional Operations page, we discussed how an OEM and / or its suppliers SHOULD use release counters in order to prevent attackers who compromise the director repository from executing rollback attacks. To further limit these attacks in this scenario, the OEM and / or its suppliers SHOULD also use the following recommendations.
+On the[Exceptional Operations](https://github.com/uptane/deployment-considerations/blob/master/exceptional_operations.md) page, we discussed how an OEM and / or its suppliers SHOULD use release counters in order to prevent rollback attacks should the Director repository be compromised. To further limit these attacks in this scenario, the OEM and / or its suppliers SHOULD also use the following recommendations.
```

Comment:
```suggestion
On the [Exceptional Operations](https://github.com/uptane/deployment-considerations/blob/master/exceptional_operations.md) page, we discuss how an OEM and/or its suppliers SHOULD use release counters in order to prevent rollback attacks if the Director repository is compromised. To further limit these attacks in this scenario, the OEM and/or its suppliers SHOULD also use the following recommendations.
```